pycoal/environment.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import coal
+
+class EnvironmentalCorrelation:
+
+    def correlateMiningEnvironment(environmentFilename, miningFilename, outputFilename):
+
+        """
+        Correlate environmental impact GIS data with classified mines and save the result to a file.
+        """
+
+        # define function to select the environmental impact layer
+        def impactLayer(environmentDataset):
```

Comment:
Is it just me or has the formatting/indent changed here from 4 space/tab to 8 space/2 tab?